modules/nlu/src/backend/api.ts

```diff
@@ -68,7 +69,9 @@ export default async (bp: typeof sdk, state: NLUState) => {
     }
 
     try {
-      const nlu = await state.nluByBot[botId].engine.predict(value.text, value.contexts)
+      let nlu = await state.nluByBot[botId].engine.predict(value.text, value.contexts)
+      // TODO: call converse API instead of calling legacy election by hand
```

Comment:
There could be a req param `includeLegacy` or something like that

Reply:
Also realized that this will impact `cross-validation` up there as well. We might want to include a call to this legacy election in cross validation code

Reply:
Sure, it will be slower but don't you think it's weird to call ```predict``` and ```legacyElectionPipeline``` back to back?

You would prefer something like this?

```
const { includeLegacy } = req.params
if (includeLegacy) {
    nlu = legacyElectionPipeline(nlu);
} else {
    nlu = newElectionPipeline(nlu) // the one called in the intent-is condition
}
```
+

Sure, I'll check for ```cross-validation```.